src/components/loginPanel/helper.js

```diff
@@ -0,0 +1,31 @@
+export const initialFormValues = {
+    email: "",
+    password: "",
+  };
+  export const testLogin = {
+    email: "test@gmail.com",
+    password: "Test123@",
+  };
+  
+  export const validateInput=(values)=>{
+    const errors = {};
+
+    const passRegex =
```

Comment:
convention is to use `UPPER_SNAKE_CASE` for RegExp. And all the regular expressions can be moved to a separate file